tests/test_models.py

```diff
@@ -90,6 +90,42 @@ def test_string_representation(self):
         assert expected_repr == course_skill.__repr__()
 
 
+@mark.django_db
+class TestXBlockSkills(TestCase):
+    """
+    Tests for the ``XBlockSkills`` model.
+    """
+
+    def test_xblock_skill_string_representation(self):
+        """
+        Test the string representation of the XBlockSkills model.
+        """
+        xblock_skill_through = factories.XBlockSkillThroughFactory()
+        xblock_skills = xblock_skill_through.xblock
+        expected_str = '<XBlockSkills usage_key="{}">'.format(
+            xblock_skills.usage_key
+        )
+        expected_repr = '<XBlockSkills id="{0}">'.format(xblock_skills.id)
+
+        assert expected_str == xblock_skills.__str__()
+        assert expected_repr == xblock_skills.__repr__()
```

Comment:
Would it make more sense to test `XBlockSkills` using `XBlockSkillsFactory`, this way we can tests the most granular components instead of testing the association? What do you think?